AwaMelvine/fast-food-fast
# #160747738 Improve the responsiveness of the UI

#### What does this PR do?
Improves the responsiveness of the UI template pages.

#### Description of Task to be completed?
- Split the CSS code into multiple files
- Changed colors of buttons to blend with the overall theme
- Make tables horizontally scrollable on smaller screens
- Make the footer to cover the whole screen on devices with smaller screens

#### How should this be manually tested?
Run the UI on the browser and open the developer tools.
Click on the screens icon to view on smaller screens. Continuously
shrink the width of the small interface that appears while observing 
who the interface responds accordingly.

#### What are the relevant pivotal tracker stories?
[#160747738] Split CSS code into different files
[#160748067] Homepage image/intro should fill screen height
[#160737399] Change the button colors to match the overall theme of the site
[#160747595] Fix footer alignment and tables display issues on mobile
[#160775411] Make items on menu page to be 3 per row and centralized
[#160747818] Improve tables styling and buttons responsiveness
[#160749526] Use smaller font sizes for email addresses listed on the tables
